config/tips.txt

```diff
@@ -31,56 +34,63 @@ As a Medical Doctor, you must target the correct limb and be on help intent when
-As a Security Officer, you should communicate and coordinate with your fellow officers using the security channel :s to avoid confusion.
+As the Warden, you can implant criminals you suspect might re-offend with devices that will track their location and allow you to remotely inject them with disabling chemicals. 
+As a Security Officer, communicate and coordinate with your fellow officers using the security channel :s to avoid confusion.
 As a Security Officer, you can use handcuffs on orange prisoner shoes to turn them into cuffed shoes. Anyone who wears these will be forced to walk.
 As a Security Officer, your sechuds or HUDsunglasses can not only see what job someone works in and their criminal status, but also if they are loyalty implanted or not. Use this to your advantage in a revolution to definitively tell who is on your side!
 As a Security Officer, loyalty implants can only prevent someone from being turned into a cultist: unlike revolutionaries, it will not de-cult them if they have already been converted.
-As a Security Officer, examining someone while wearing sechuds or HUDsunglasses will let you set their arrest level, which will cause beepsky or any other security bots to chase after them.
+As a Security Officer, examining someone while wearing sechuds or HUDsunglasses will let you set their arrest level, which will cause Beepsky or any other security bots to chase after them.
 As a Security Officer, implanting a gang member the first time will deconvert them, but destroy the implant. You must implant them a second time to protect them from further conversion attempts. Keep in mind that gang members have ways to destroy implants in people!
 As the Detective, people leave fingerprints everywhere and on everything. With the exception of white latex, gloves will hide them. All is not lost, however, as gloves leave fibers specific to their kind such as black or nitrile, pointing to a general department.
 As the Detective, you can use your forensics scanner from a distance.
 As the Lawyer, try to negotiate with the Warden if sentences seem too high for the crime.
+As the Lawyer, you can try to convince the captain and Head of Security to hold trials for prisoners in the courtroom.
 As the Head of Personnel, you are not higher ranking than other heads of staff, even though you are expected to take the Captain's place first should he go missing. If the situation seems too rough for you, consider allowing another head to become temporary Captain.
 As the Head of Personnel, you are often just as large a target as the Captain because of the potential power your ID and computer can hand out.
 As the Mime, you can place invisible walls that last for a short time to block other people. You can use it in a pinch to delay your pursuer.
 As the Clown, if you lose your banana peel, you can still slip people with your PDA! Honk!
 As the Clown, you have the clumsiness genetic defect, and will misfire guns if you try to fire them.
```

Comment:
This one is barely a useful tip/trick. 

Also its funnier when new clowns encounter clumsiness unaware